transport/owr_transport_agent.c

```diff
@@ -912,6 +914,7 @@ static gboolean add_session(GHashTable *args)
         g_signal_emit_by_name(priv->rtpbin, "get-internal-session", stream_id, &rtp_session);
         g_object_set_data(rtp_session, "session_id", GUINT_TO_POINTER(stream_id));
         g_signal_connect_after(rtp_session, "on-sending-rtcp", G_CALLBACK(on_sending_rtcp), transport_agent);
+        g_signal_connect_after(rtp_session, "on-feedback-rtcp", G_CALLBACK(on_feedback_rtcp), NULL);
```

Comment:
Or use on-receiving-rtcp here, otherwise this will only be called for feedback and not for SR/RR